official/guides/create-carrier-curls/quick.sh

```diff
@@ -0,0 +1,13 @@
+curl -X POST https://www.easypost.com/api/v2/carrier_accounts \
+  -u "$EASYPOST_API_KEY": \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "type": "QuickAccount"
+  "description: "QuickCourierAccount",
+    "billToEasyPost": false,
+      "credentials": {
+        "username": "<username>",
+        "password": "<password>"
+      },
+      "test_credentials": {},
+}'
```

Comment:
Please add a new line at the end of the file
```suggestion
}'

```